<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabs">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;catalog&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="catalogTab">
       <attribute name="title">
        <string>üìö –ö–∞—Ç–∞–ª–æ–≥</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="catalogLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="filterLayout">
            <item>
             <widget class="QLabel" name="filterLabel">
              <property name="text">
               <string>–°–µ–∫—Ü—ñ—è:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cbSection"/>
            </item>
            <item>
             <widget class="QLineEdit" name="searchBar"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="booksTable"/>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteBook">
            <property name="text">
             <string>–í–∏–¥–∞–ª–∏—Ç–∏ –∫–Ω–∏–≥—É</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="sectionsTab">
       <attribute name="title">
        <string>üìÇ –°–µ–∫—Ü—ñ—ó</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="sectionsLayout">
          <item>
           <widget class="QTableView" name="SectionsTable"/>
          </item>
          <item>
           <widget class="QPushButton" name="DeleteSection">
            <property name="text">
             <string>–í–∏–¥–∞–ª–∏—Ç–∏ —Å–µ–∫—Ü—ñ—é</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="usersTab">
       <attribute name="title">
        <string>üë§ –ö–æ—Ä–∏—Å—Ç—É–≤–∞—á—ñ</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="usersLayout">
          <item>
           <widget class="QTableView" name="usersTable"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="buttonsLayout">
            <item>
             <widget class="QPushButton" name="deleteUser">
              <property name="text">
               <string>–í–∏–¥–∞–ª–∏—Ç–∏ –∫–æ—Ä–∏—Å—Ç—É–≤–∞—á–∞</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="createUser">
              <property name="text">
               <string>–î–æ–¥–∞—Ç–∏ –∫–æ—Ä–∏—Å—Ç—É–≤–∞—á–∞</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="title">
        <string>‚öôÔ∏è –ù–∞–ª–∞—à—Ç—É–≤–∞–Ω–Ω—è</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <layout class="QFormLayout" name="settingsLayout">
          <property name="leftMargin">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lbLanguage">
            <property name="text">
             <string>–º–æ–≤–∞</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="cbLanguage"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="helpTab">
       <attribute name="title">
        <string>‚ùì–î–æ–ø–æ–º—ñ–∂–Ω–∞ —ñ–Ω—Ñ–æ—Ä–º–∞—Ü—ñ—è</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <layout class="QVBoxLayout" name="helpLayout">
          <item>
           <widget class="QTextBrowser" name="help"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>–§–∞–π–ª</string>
    </property>
    <addaction name="newBookAction"/>
    <addaction name="separator"/>
    <addaction name="newSectionAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newBookAction">
   <property name="text">
    <string>&amp;–ù–æ–≤–∞ –∫–Ω–∏–≥–∞</string>
   </property>
  </action>
  <action name="newSectionAction">
   <property name="text">
    <string>–ù–æ–≤–∞ &amp;—Å–µ–∫—Ü—ñ—è</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="text">
    <string>&amp;–í–∏—Ö—ñ–¥</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
